### DISCUSSION TITLE
useFetch how to use proxyAgent

### DISCUSSION THREAD

**Discussion by automa-wiki** [nuxt/nuxt#29916] [Questions]
### Describe the feature

useFetch how to use proxyAgent

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [x] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [x] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by huang-julien** [comment]
Hi :wave: you can probably try using `dispatcher`. See the PR for ofetch docs [here ](https://github.com/unjs/ofetch/pull/389/files)
